yellowbrick/classifier/rocauc.py

```diff
@@ -192,6 +197,27 @@ def __init__(
         # Set the visual parameters for ROCAUC
         self.set_params(micro=micro, macro=macro, per_class=per_class)
 
+    def fit(self, X, y=None):
+        """
+        Fit the classification model.
+        """
+        # The target determines what kind of estimator is fit
+        ttype = type_of_target(y)
+        if ttype.startswith(MULTICLASS):
+            self.target_type_ = MULTICLASS
+        elif ttype.startswith(BINARY):
+            self.target_type_ = BINARY
```

Comment:
I think it's a good idea to store the discovered target type in a learned attribute, but this also needs to be added to the docstring, which will also help users diagnose issues with target types that are are not binary or multiclass.